core/glrender/src/main/java/edu/cmu/cs/dennisc/render/gl/imp/adapters/GlrDisc.java

```diff
@@ -100,7 +138,7 @@ protected void pickGeometry(PickContext pc, boolean isSubElementRequired) {
       name = -1;
     }
     pc.gl.glPushName(name);
-    glDisc(pc);
+    glDisc(pc, false);
```

Comment:
Is this supposed to be `glDisc(pc, true);` ?

Reply:
This in pickGeometry that is used to determine interactions with the scene, so the full rendering is not required. In renderGeometry the isTextureEnabled does get checked and used (and is probably no longer needed above on line 125)